pyaerocom/io/readungridded.py

```diff
@@ -27,6 +27,7 @@
 from pyaerocom.io.read_eea_aqerep import ReadEEAAQEREP
 from pyaerocom.io.read_eea_aqerep_v2 import ReadEEAAQEREP_V2
 from pyaerocom.io.read_marcopolo import ReadMarcoPolo
+from pyaerocom.plugins.ipcforests.reader import ReadIPCForest
```

Comment:
`ReadIPCForest` was implemented as a plugin, so it does not need to be imported here, but declared as an entry point.

Reply:
There are no working plugins as we have experienced with the MEP reader. This reader just copies the way MEP was implemented.
I kept them in the plugins sub directory to mark them as needing additional work. This is also referenced in the issue #842